src/index.d.ts

```diff
@@ -82,6 +82,8 @@ export interface ForceGraphGenericInstance<ChainableInstance> {
   linkColor(colorAccessor: LinkAccessor<string>): ChainableInstance;
   linkAutoColorBy(): LinkAccessor<string | null>;
   linkAutoColorBy(colorByAccessor: LinkAccessor<string | null>): ChainableInstance;
+  linkLineDash(): LinkAccessor<number[]>;
```

Comment:
Since it can be `null` the generic should be `<number[] | null>`.

Reply:
Done